guides/source/upgrading_ruby_on_rails.md

```diff
@@ -85,13 +85,12 @@ Rails 6.1. You are encouraged to enable `config.force_ssl` to enforce HTTPS
 connections throughout your application. If you need to exempt certain endpoints
 from redirection, you can use `config.ssl_options` to configure that behavior.
 
-### Purpose in signed or encrypted cookie is now embedded within cookies
+### Purpose and expiry metadata is now embedded inside signed and encrypted cookies for increased security
 
-To improve security, Rails embeds the purpose information in encrypted or signed cookies value.
```

Comment:
This sentence seemed basically repeating the same thing as the new sub-heading.

Reply:
Lets retain this, it says why we do it.

Reply:
Lets retain this to maintain the flow of sentences below

Reply:
@vipulnsward Retained this sentence with some modifications made to it for consistency.